typescript/src/entities/variation.ts

```diff
@@ -23,6 +23,18 @@ export class Variation extends Entity {
   @Variation.property("cost")
   public cost?: number;
 
+  @Variation.property("quantity")
+  public quantity?: number;
+
+  @Variation.property("onceOffCost")
+  public onceOffCost?: number;
+
+  @Variation.property("unitCost")
+  public unitCost?: number;
+
+  @Variation.property("unitCostTotal")
+  public unitCostTotal?: number;
+
```

Comment:
I've noticed in this file you've set 'value' to string | null; I think it can take number as well. Sometimes a value can be an id of an option I think.

Reply:
`value` might contain a number, but as far as i can tell it's type is always `string`

Reply:
@Amtron 

@DanielCollins is correct about this, value can be showed as number, string or ids but in terms of type they are string